code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -1250,6 +1252,101 @@
 
 	return ..()
 
+/datum/hive_status/corrupted/renegade
+	name = "Renegade Hive"
+	reporting_id = "renegade"
+	hivenumber = XENO_HIVE_RENEGADE
+	prefix = "Renegade "
+	color = "#9c7a4d"
+	ui_color ="#80705c"
+
+	dynamic_evolution = FALSE
+	allow_no_queen_actions = TRUE
+	allow_queen_evolve = FALSE
+	ignore_slots = TRUE
+	latejoin_burrowed = FALSE
+
+/datum/hive_status/corrupted/renegade/New()
+	. = ..()
+	hive_structures_limit[XENO_STRUCTURE_EGGMORPH] = 0
+	hive_structures_limit[XENO_STRUCTURE_EVOPOD] = 0
+	allies[FACTION_MARINE] = TRUE
+
+/datum/hive_status/corrupted/renegade/faction_is_ally(faction, ignore_queen_check = TRUE)
+	return . = ..()
```

Comment:
```suggestion
	return ..()
```
But also this currently entire proc at this moment can just be removed since all it does is call parent.

Reply:
Parent has ignore_queen_check = FALSE, so alliance doesn't work because they have no queen

Reply:
This is not a very intuitive way to do this but I guess.